src/vs/workbench/contrib/chat/browser/chatPasteProviders.ts

```diff
@@ -111,9 +120,35 @@ export class PasteImageProvider implements DocumentPasteEditProvider {
 		const edit = createCustomPasteEdit(model, scaledImageContext, mimeType, this.kind, localize('pastedImageAttachment', 'Pasted Image Attachment'), this.chatWidgetService);
 		return createEditSession(edit);
 	}
+
+	private async createFileForMedia(
+		dataTransfer: Uint8Array,
+		mimeType: string,
+	): Promise<URI | undefined> {
+		const workspaceFolder = this.workspaceService.getWorkspace().folders[0];
+		if (!workspaceFolder) {
+			return;
+		}
+
+		const imagesFolder = joinPath(workspaceFolder.uri, '.chat-images');
```

Comment:
Seems unexpected that I'd paste an image and it would show up in my workspace, am I expected to check it in? How about in a temp folder, or vscode's workspace data folder, with some logic to rotate them or remove them after some amount of time?

Reply:
If we were going to do this it should probably be under `.vscode` or have a name that ties it to vscode in some way

Reply:
Yeah that's probably a good idea, I was going to ask in standup if folks had thoughts on this - we do something similar with markdown but it adds the file under the same folder of the current open file.